wordpress-mobile/WordPress-iOS
# Include Gutenberg without submodules

Uses the changes in https://github.com/wordpress-mobile/gutenberg-mobile/pull/255 to avoid the Gutenberg submodule and fetch the pod via git.

To test:
- Run `pod install`
- Make sure the app builds and the existing integration works as it used to

For local development:
- If metro is running `yarn start`, the app should run Gutenberg from there instead of the bundled JS

For bridge development:
- Edit the `Podfile` and change the Gutenberg line to point to your local copy (e.g. `pod 'Gutenberg', :path => '../gutenberg-mobile'`
- Run `pod install` and develop normally
- Make sure you don't commit the changes to the `Podfile`
